packages/markdown-docx/src/ToOOXMLVisitor/index.js

```diff
@@ -59,6 +63,49 @@ class ToOOXMLVisitor {
         return node.$class;
     }
 
+    /**
+     * Generates the OOXML for text and code ciceromark nodes.
+     *
+     * @param {string}  value           Text value of the node
+     * @param {Array}   nodeProperties  Properties of the node
+     * @param {boolean} calledByCode    Is function called by code node or not
+     * @returns {string} Generated OOXML
+     */
+    generateTextOrCodeOOXML(value, nodeProperties, calledByCode = false) {
+        let propertyTag = '';
+        if (calledByCode) {
+            propertyTag = CODE_PROPERTIES_RULE();
+        }
+        let isLinkPropertyPresent = false;
+        for (const property of nodeProperties) {
+            if (property === TRANSFORMED_NODES.emphasize) {
+                propertyTag += EMPHASIS_RULE();
+            } else if (property === TRANSFORMED_NODES.strong) {
+                propertyTag += STRONG_RULE();
+            } else if (property === TRANSFORMED_NODES.link) {
+                isLinkPropertyPresent = true;
+                propertyTag += LINK_PROPERTY_RULE();
+            }
+        }
+        if (propertyTag) {
+            propertyTag = TEXT_STYLES_RULE(propertyTag);
+        }
+
+        let textValueTag = TEXT_RULE(value);
+
+        let tag = TEXT_WRAPPER_RULE(propertyTag, textValueTag);
+
+        if (isLinkPropertyPresent) {
+            // Two relationships for numbering and style are already present
+            // and since we need to accommodate for link styles as well, we need a unique ID
+            // to represent them. Hence, 2 is added to offset the enumeration of `rId`.
```

Comment:
This comment would go in that constants file now. Ensure that you link the filename too.